### PR TITLE
feat: add draft of resources docs

### PR DESCRIPTION
**Pull request by cmoog**
cc @kylecarbs @ashumz 

https://codercom-9fs9sbo2t.vercel.app/docs/admin/resources

### CODE REVIEW

**Comment by cmoog** [comment]
Adding more detailed content / reference docs at the bottom.

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+  "editor.rulers": [80]
```
Comment (kylecarbs) [inline .vscode/settings.json:2]: 🧠

File: admin/resources.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
+---
+title: Compute Resources
+description: Learn the unique compute resource management capabilities in Coder.
+---
+
+## Overview
```
Comment (kylecarbs) [inline admin/resources.md:6]: Not certain this overview header needs to exist... kinda implicit

File: admin/resources.md [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+Consider the case where a project's compilation is parallelizable up to 16 CPU
+cores. To provide a more tolerable build time, each developer is given a 16
+CPU core laptop. During builds, the machine sees 100% utilization. But, notice
+that the machine is underutilized a vast majority of a typical workday.
+Only during the few minutes of compilation are the resources
+utilized.
+
+![resources-nonshared.svg](../assets/resources-old.svg)
+
+### Shared Resources With Coder Workspaces on Your Cloud Infrastructure
```
Comment (kylecarbs) [inline admin/resources.md:29]: Feels like this is quite verbose but minor nitpick

File: admin/resources.md [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+access to CPU, instead they have **short periods of peak usage** during builds and
+compilations, followed by **long periods of relative idling**.
+
+Build and compilation performance directly impacts the development
+experience of a project– faster builds mean faster iteration cycles,
+leading to greater development velocity. But, traditional approaches to
+providing developers with more hardware for computationally intensive
+compilations leads to wasted resources and sunk costs.  
+
+### With Expensive Laptops / Desktops
```
Comment (kylecarbs) [inline admin/resources.md:18]: This should be h2

File: admin/resources.md (hunk 3 above)
Comment (kylecarbs) [reply admin/resources.md:29]: Should also be h2

File: admin/resources.md [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+
+**What happens during resource contention?**
+
+If resources on the underlying node become contented, CPU cycles are shared
+on a weighted basis relative to the resource request of the Coder Workspace.
+But, the nature of developer workflows makes resource contention far less
+common than in other server workloads, given that it only occurs when several
+users are simultaneously performing an infrequent compilation task.
+
+## Configuring Shared Resources in Coder
```
Comment (kylecarbs) [inline admin/resources.md:53]: h3

File: admin/resources.md [hunk 6]
```diff
@@ -0,0 +1,64 @@
...
+has access to the proper resources during peak load– providing a
+performant experience when needed, with less waste during periods of idling.
+
+<!-- Notice how each developer has access to greater
+resources during peak load– providing a superior experience when needed,
+with less waste when not. -->
+
+![resources-shared.svg](../assets/resources-new.svg)
+
+**What happens during resource contention?**
```
Comment (kylecarbs) [inline admin/resources.md:45]: h3 boi

**Review by kylecarbs — COMMENTED** [review]
I'll update the graphs... most of these are minor nitpicks.

Overall, massive massive contribution 🧠

**Comment by cmoog** [comment]
Fixed the trash header structure and use the `Environment` product nomenclature.

**Review by kylecarbs — APPROVED** [review]

**Comment by bpmct** [comment]
@cmoog thanks for including me in this, Ashley and I will be using this as a resource.

please feel free to tag me in future stuff as well.

**Pull request merged**